db/schema.rb

```diff
@@ -306,6 +306,7 @@
     t.index ["coin_id"], name: "index_feed_sources_on_coin_id"
     t.index ["feed_type"], name: "index_feed_sources_on_feed_type"
     t.index ["feed_url"], name: "index_feed_sources_on_feed_url", unique: true
+    t.index ["id", "is_active", "feed_type"], name: "compound_index"
```

Comment:
Hm this looks new, what's the background?

Reply:
Looks like it's from a while ago but the schema.db was never updated.

https://github.com/coinfi/coinfi/blob/develop/db/migrate/20180928070531_add_is_active_index_to_feed_sources.rb

Reply:
Huh weird, the line below (310) should be the output for that migration though...

Let's go ahead and remove still unless I'm missing something?

Reply:
Ah, you're right. I think this comes from a compound index I manually added when trying to optimize news items. :cry: